python/baseline/services.py

```diff
@@ -238,8 +195,11 @@ def predict(self, tokens, preproc=None):
         """
         if preproc is not None:
             logger.warning("Warning: Passing `preproc` to `ClassifierService.predict` is deprecated.")
-        tokens_seq, mxlen, mxwlen = self.batch_input(tokens)
-        self.set_vectorizer_lens(mxlen, mxwlen)
+        tokens_seq = self.batch_input(tokens)
```

Comment:
I know this is the name we used before but this would be clearer if the name was something like `tokens_batch` given that it is returned form the batch_input function